tools/experiments/experiments.js

```diff
@@ -58,12 +58,22 @@ const BETA_CHANNEL_ID = 'beta-channel';
 const RTV_CHANNEL_ID = 'rtv-channel';
 
 /**
- * The different states of the AMP_CANARY cookie.
+ * The different states of the __Host-AMP_OPT_IN cookie.
  */
-const AMP_CANARY_COOKIE = {
+const AMP_OPT_IN_COOKIE = {
   DISABLED: '0',
-  CANARY: '1',
-  RC: '2',
+  EXPERIMENTAL: 'experimental',
+  BETA: 'beta',
```

Comment:
Will there eventually be a value for `NIGHTLY`? If so, maybe include a TODO in the comment to acknowledge this fact?

Reply:
Done